include/slb/detail/config.hpp

```diff
@@ -27,4 +27,24 @@
 #define __has_feature(x) 0
 #endif
 
+// libstdc++ did not mark `operator()` as `noexcept` until version 8.
+#if __cpp_lib_integral_constant_callable &&                                    \
+    (!defined(__GLIBCXX__) || __has_include(<filesystem>))
+#define SLB_CPP_LIB_INTEGRAL_CONSTANT_CALLABLE 2 // available / conforming
+#else
+#define SLB_CPP_LIB_INTEGRAL_CONSTANT_CALLABLE 0 // not available
```

Comment:
This should be "`1 // available / non-conforming`" instead.

Reply:
Good catch. Thanks!